lib/logging.js

```diff
@@ -44,28 +43,28 @@ var makeTransports = function() {
         }));
     }
     if (loggerConfig.logfile) {
-        transports.push(new (winston.transports.File)({
+        transports.push(new (winston.transports.DailyRotateFile)({
             filename: loggerConfig.logfile,
             json: false,
             name: "logfile",
             level: loggerConfig.level,
             timestamp: timestampFn,
             formatter: formatterFn,
-            maxsize: loggerConfig.maxFileSizeBytes,
             maxFiles: loggerConfig.maxFiles,
+            datePattern: ".yyyy-MM-dd",
```

Comment:
I can imagine that people might want to at least configure the `datePattern`, it might be an idea to stick it in config and default it to ".yyyy-MM-dd"